helix-core/src/position.rs

```diff
@@ -74,23 +71,25 @@ pub fn coords_at_pos(text: RopeSlice, pos: usize) -> Position {
 /// not in the document in the future.
 /// See [`coords_at_pos`] for an "objective" one.
 pub fn visual_coords_at_pos(text: RopeSlice, pos: usize, tab_width: usize) -> Position {
-    let line = text.char_to_line(pos);
-
```

Comment:
This is a small stylistic nit but it's more common in the codebase to see `line` when translating a position to a line number (rather than `row`)

Reply:
Maybe this is just a language barrier but for me it was confusing because there is `row` in `Position`. In any case I'm fine with any name.